sci-misc/kaldi/kaldi-0_p20160628.ebuild

```diff
@@ -0,0 +1,133 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit toolchain-funcs cuda flag-o-matic
+
+COMMIT_ID="16e69f1aacce8ad0665d2b6666c053b1421a9e91"
+DESCRIPTION="A toolkit for speech recognition"
+HOMEPAGE="http://kaldi-asr.org/"
+SRC_URI="https://github.com/kaldi-asr/kaldi/archive/${COMMIT_ID}.tar.gz -> ${P}.tar.gz"
+
+S=${WORKDIR}/${PN}-${COMMIT_ID}/src
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE="doc test cuda speex"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	speex? ( media-libs/speex )
+	virtual/cblas
+	virtual/lapack
+	virtual/lapacke
+	<sci-misc/openfst-1.5.0
+	cuda? ( dev-util/nvidia-cuda-toolkit )"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen )"
+
+# We need write acccess /dev/nvidiactl, /dev/nvidia0 and /dev/nvidia-uvm and the portage
+# user is (usually) not in the video group
+RESTRICT="test? ( cuda? ( userpriv ) )"
```

Comment:
Please take a look into the imagemagick package how it could be solved without this use chain.